lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -100,9 +104,12 @@ def ext_delete(cursor, ext):
         return False
 
 
-def ext_create(cursor, ext):
+def ext_create(cursor, ext, schema):
     if not ext_exists(cursor, ext):
-        query = 'CREATE EXTENSION "%s"' % ext
+        query_fragments = ['CREATE EXTENSION "%s"' % ext]
```

Comment:
```
if schema:
    query += ' WITH SCHEMA "%s"' % schema
```
It looks simpler than the way with a list and we just add 2 rows instead of 5 (4 + 1 changed).

Reply:
Done